cmake/macros/macro_deal_ii_add_test.cmake

```diff
@@ -391,9 +462,9 @@ MACRO(DEAL_II_ADD_TEST _category _test_name _comparison_file)
             DEPENDS ${TEST_DEPENDENCIES_${_target}}
             )
         ENDIF()
-        SET(TEST_DEPENDENCIES_${_target} ${_test_full})
+        SET(TEST_DEPENDENCIES_${_target} ${_test_full} PARENT_SCOPE)
```

Comment:
Why did you add `PARENT_SCOPE`?

Reply:
Because I converted the macro into a function in order to use `RETURN()`.
In order to set a variable name in the (now) parent scope I have to use the `PARENT_SCOPE` keyword.